desktop/docs/development.md

```diff
@@ -0,0 +1,77 @@
+# Development
+
+At its simplest level, the WordPress.com for Desktop app uses Electron to wrap Calypso inside a native app.
+
+Electron provides all the interfacing between Chrome (the browser that is used inside Electron), and the native platform. This means we can re-use Calypso code while still providing native platform features.
+
+It is important to understand where code runs, and for this the terminology is:
+
+- Main - this is the Electron wrapper. All code for this is contained in `desktop`
+- Renderer - this is the Chrome browser and is where Calypso runs
+
+We use Electron's [IPC](https://github.com/atom/electron/blob/master/docs/api/ipc-main.md) mechanism to send data between the main process and the renderer.
+
+## Starting the app
+
+### How does it work?
+
+It's a fairly complicated process so buckle up. Note that *(main)* and *(renderer)* will be added to show where the code actually runs.
+
+For clarity, all file and folder locations are relative to the root of the Calypso monorepo.
+
+- *(main)* Electron looks at the `main` item in `desktop/package.json` - this is the boot file, and refers to `client/desktop/index.js`
+- *(main)* `client/desktop/index.js` sets up the environment in `client/desktop/env.js` - this includes Node paths for Calypso
+- *(main)* Various [app handlers](../../client/desktop/app-handlers/README.md) are loaded from `client/desktop/app-handlers` - these are bits of code that run before the main window opens
+- *(main)* A Calypso server is started drectly from Electron's Node process in `client/desktop/server.js`. The server is customized to serve files from the following directories:
```

Comment:
It'd be good to note here that the "Calypso" server is an Express.js HTTP server that serves files for the Renderer process.